doc/source/regression/wp.md

```diff
@@ -357,22 +353,22 @@ x = [1, 2, 3, 4, 5, 6, 7, 8]
 wp = pywt.WaveletPacket(data=x, wavelet='db1', mode='symmetric')
 ```
 
-1. At first the wp's attribute `a` is `None`:
```

Comment:
I decided to remove list notation from the markdown files because in my opinion it creates confusing and non-semantic HTML. What happens is that several list elements get created in the HTML where there should only be one list element. It's unfortunate but as far as I can tell, there's no good way to put notebook cells in a list. They are block level and cannot be inlined or included within any other node.